### PR TITLE
Add skeleton for RsFileSink with the new structure

### PR DESCRIPTION
**Pull request by luisbg**

### CODE REVIEW

**Comment by sdroege** [comment]
You forgot to git add rssink.rs :)

File: src/rssink.c [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+{
+  return GST_URI_SINK;
+}
+
+static const gchar *const *
+gst_rs_sink_uri_get_protocols (GType type)
+{
+  static const gchar *protocols[] = { "file", NULL };
+
+  return protocols;
```
Comment (sdroege) [inline src/rssink.c]: This should also look into the hash table

**Comment by luisbg** [comment]
Fixed :)

**Comment by sdroege** [comment]
Merged

**Pull request closed without merging**